about.html

```diff
@@ -200,7 +200,7 @@ <h2 class="logo_dtitle">Digital Youth GTE 2023 Logo</h2>
           <div class="logo_img">
             <a href="#">
               <img
-                src="/SVG/gte-ahmedabad-2023.jpg"
+                src="SVG/gte-ahmedabad-2023.jpg"
                 alt="logo of GTE Global summit 2023"
                 height="200px"
             /></a>
```

Comment:
- [ ] Make sure that your site looks like the provided design, kindly remove the below section so your site is as the provided design

- Kindly remove this section:
![Screenshot_20221029_010005](https://user-images.githubusercontent.com/25227022/198752142-2c4f54fc-c20d-4305-a83d-7172f17ff21d.png)

Reply:
done! thankyou